diff --git a/core/jni/fd_utils-inl.h b/core/jni/fd_utils-inl.h
index df880800..8c3af7d9 100644
--- a/core/jni/fd_utils-inl.h
+++ b/core/jni/fd_utils-inl.h
@@ -59,6 +59,7 @@ static const char* kPathWhitelist[] = {
   "/sys/kernel/debug/tracing/trace_marker",
   "/system/framework/framework-res.apk",
   "/dev/urandom",
+  "/proc/ged",
   "/dev/ion",
   "@netlink@",
   "/system/framework/org.cyanogenmod.platform-res.apk",
